backend/testfiles/execution/internal.tests

```diff
@@ -3,9 +3,21 @@
 // ---------------
 // Misc
 // ---------------
+[tests.misc]
```

Comment:
Because this test turned out to be more than one line, I was sort of forced to add a [tests.] group and the [test] heading to the other 'misc' tests. Am I missing some way around this?

Reply:
That's how you do it